build-system/server/new-server/transforms/utilities/script.ts

```diff
@@ -76,12 +76,12 @@ export function toExtension(url: URL, extension: string): URL {
  * This is a temporary measure to allow for a relaxed parsing of our
  * fixture files' src urls before they are all fixed accordingly.
  */
-export function tryGetUrl(src: string, port: number = 8000): URL {
+export function tryGetUrl(src: string, host: string = '0.0.0.0', port: number = 8000): URL {
```

Comment:
Depending on the caller, this is unneeded. If you already know the absolute path, just use `/path/to/script.mjs`. That means the returned URL will always be on whatever host is currently being requested, without you having to know about it.

Reply:
went ahead and applied your suggestion. PTAL

Reply:
This is a separate concern. What I mean here is that the caller of `tryGetUrl` probably doesn't need to call it at all. Instead, they can just use an host-relative absolute path (eg, `/path/to/script.js`).